tuna/worker_interface.py

```diff
@@ -89,54 +65,49 @@ def __init__(self, **kwargs):
 
     allowed_keys = set([
         'machine', 'gpu_id', 'num_procs', 'barred', 'bar_lock', 'envmt',
-        'reset_interval', 'fin_steps', 'fin_infile', 'fin_outfile', 'job_queue',
-        'queue_lock', 'label', 'fetch_state', 'docker_name', 'end_jobs',
-        'config_type', 'dynamic_solvers_only', 'session_id'
+        'reset_interval', 'fin_steps', 'job_queue', 'queue_lock', 'label',
```

Comment:
can we rename or move fin_steps somewhere else? its really miopen specific.

Reply:
That's going to take more work, so it will be addressed in the next iteration. This also necessitates changes to the job queue.